site/components/header/small-screen-nav.js

```diff
@@ -29,6 +29,9 @@ export default class SmallScreenNav extends React.Component {
   render() {
     const { navOpen } = this.state;
     const navTabIndex = navOpen ? 0 : -1;
+    const { stateNavigator } = this.context;
+    const contactUsLink = `${stateNavigator.historyManager.getHref(stateNavigator.getNavigationLink('homePage'))}#ContactUs`;
```

Comment:
@grahammendick Do you want to take that question from @JoeStanton. I think this is not possible if the user is on another page (e.g. about-us).